_security-plugin/access-control/document-level-security.md

```diff
@@ -124,9 +124,9 @@ PUT _plugins/_security/api/roles/abac
   }]
 }
 ```
-## Use term-level lookup queries (TLQs) with DLS 
+## Use term lookup queries (TLQs) with DLS 
```

Comment:
Because TLQs is defined earlier, we only need to use the abbreviation here: "Use TLQs with DLS".